frontend/src/lib/lemon-ui/LemonRow/LemonRow.tsx

```diff
@@ -97,23 +94,9 @@ export const LemonRow = React.forwardRef(function LemonRowInternal<T extends key
         },
         <>
             <div className="LemonRow__main-area">
-                {icon && (
-                    <span
-                        className={clsx(
-                            'LemonRow__icon',
-                            'LemonRow__icon--prefix',
-                            relaxedIconWidth && 'LemonRow__icon--relaxed-width'
```

Comment:
Why have these changes been necessary?

Reply:
This was just dead code, noticed when fixing a visual regression f5b3618f3942d0ee16236e915867c660fb277e6e